config.js

```diff
@@ -60,6 +60,18 @@ module.exports = {
 
     // The number of transactions we buffer before we publish to the Cloud Trace
     // API, unless we hit `flushDelaySeconds` first.
-    bufferSize: 1000
+    bufferSize: 1000,
+
+    // If true, the trace agent will attempt to publish any buffered traces if an
+    // uncaught exception occurs. This will result in the error being silenced. If
+    // you want your application to exit on uncaught exception you will have to
+    // add your own listener to the 'uncaughtException' event on the process.
+    publishOnUncaught: false,
```

Comment:
I am not personally a fan of using strings as enums in configuration. If we want to reduce the configuration surface area, I would prefer offering a single `flushAndExit` boolean. If people are going to be recovering and silencing the errors then they would likely not care that we don't flush when the process would otherwise have gone down. Would this be acceptable?